rules/linux/auditd/lnx_auditd_disable_aslr_protection.yml

```
@@ -8,6 +8,7 @@ description: |
     A successful use of this flag can reduce the effectiveness of ASLR and make memory corruption
     attacks more reliable.
 references:
+    - https://github.com/CheraghiMilad/bypass-Neo23x0-auditd-config/blob/main/personality-syscall/README.md
```

```suggestion
    - https://github.com/CheraghiMilad/bypass-Neo23x0-auditd-config/blob/f1c478a37911a5447d5ffcd580f22b167bf3df14/personality-syscall/README.md
```
Hi,
it is better to use permalink with github references